src/lib/autoUpdater.js

```diff
@@ -0,0 +1,37 @@
+const GhReleases = require('electron-gh-releases');
```

Comment:
I was just checking and this module is deprecated. They are recommending using [update.electronjs.org](https://github.com/electron/update.electronjs.org)
I checked the requirements and we have everything, including code signed.

Reply:
Without adding any options and calling `require('update-electron-app')();`, it updated the app to version 1.6.2 for some reason

Reply:
This is really weird. The feed url we construct is 
https://update.electronjs.org/headsetapp/headset-electron/darwin-x64/3.0.0

When you open it, it shows 1.6.2 as the next update.

Reply:
oh I know why. The `zip` file must have the word "mac" in it. common...